app/validation/validator.py

```diff
@@ -281,6 +282,15 @@ def _validate_answer_actions(self, answer):
 
         return errors
 
+    def _ensure_answer_labels_and_values_match(self, answer):
+        errors = []
+        for option in answer.get('options', []):
+            if isinstance(option['label'], str) and option['label'] != option['value']:
+                errors.append(
+                    (self._error_message(f'`Found mismatching answer value(s) and label(s) for {answer["id"]}`')))
```

Comment:
We also have the answer label here, so it would be useful to include that in the error message, and then we can also then drop the `(s)`'s

Reply:
Updated this